.github/workflows/reusable_github_release.yml

```
@@ -0,0 +1,124 @@
+name: Reusable Release Workflow
+
+on:
+  workflow_call:
+    inputs:
+      tag:
+        description: 'Tag to release'
+        required: true
+        type: string
+      description:
+        description: 'Release description'
+        required: true
+        type: string
+      prerelease:
+        description: 'Whether prerelease is enabled'
+        required: true
+        type: boolean
+      changelog_path:
+        description: 'Use to link to CHANGELOG.md'
```

```suggestion
        description: 'The filepath of the changelog file'
```

What format should be followed for the path? Is it relative to the repo root or relative to the pipeline folder?

---

The path is the absolute link to the CHANGELOG.md of the repo (e.g https://github.com/babylonlabs-io/babylon/blob/main/CHANGELOG.md)